Rakefile

```diff
@@ -2,6 +2,9 @@
 
 $: << File.join(File.dirname(__FILE__), "lib")
 
+# TODO: (colin) temporary fix to access logstash-core
```

Comment:
not sure but for now it works - I don't foresee a situation where logstash-core will not be physically in this path. this could happen if it is move in another repo for example but this is not planned.